src/msg/simple/Pipe.cc

```diff
@@ -427,9 +427,6 @@ int Pipe::accept()
       goto fail_unlocked;
     }
 
-    // sanitize features
-    connect.features = connect.features;
-
```

Comment:
@gregsfortytwo i did check the history as this statement is quite weird from the first glance. they were meant to do something. the commit Sage changed to this is ff1959282826ae6acd7134e1b1ede74ffd1cc04a. and `ceph_sanitize_features()` has gone. i see no reason to keep the `foo = foo` around anymore.